src/containers/cityDetails.container.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { cityActions ,forecastActions} from '../actions';
+import { connect } from 'react-redux';
+
+class CityDetailsContainer extends React.Component {
+    componentDidMount(){
+        const {getCity,getForecast} = this.props;
```

Comment:
Here you can get place_id like : `{getCity,getForecast, match: {params: place_id: placeId}}`

Reply:
+